ledger/store/trackerdb/pebbledbdriver/pebbledriver.go

```diff
@@ -345,6 +350,12 @@ func (s *trackerStore) TransactionContext(ctx context.Context, fn trackerdb.Tran
 	return err
 }
 
+// TransactionContextWithRetryClearFn implements trackerdb.Store.
+// It currently ignores rollbackFn.
+func (s *trackerStore) TransactionContextWithRetryClearFn(ctx context.Context, fn trackerdb.TransactionFn, rollbackFn trackerdb.RetryClearFn) error {
```

Comment:
Is this actually okay or do we need a TODO? (Longer term, debating whether this code needs to be evicted as it remains unused).